src/Aspire.Hosting.Azure/Provisioning/Provisioners/BicepProvisioner.cs

```diff
@@ -420,12 +429,16 @@ private static async Task<bool> ExecuteCommand(ProcessSpec processSpec)
         return null;
     }
 
-    internal static string GetChecksum(AzureBicepResource resource, JsonObject parameters)
+    internal static string GetChecksum(AzureBicepResource resource, JsonObject parameters, JsonObject? scope = null)
```

Comment:
Do we ever call this without setting `scope`? Can it just be:

```suggestion
    internal static string GetChecksum(AzureBicepResource resource, JsonObject parameters, JsonObject? scope)
```

Reply:
Yes, the only place where we didn't was in the test cases but we can update those.